packages/documentation/pages/usage/components/popover.vue

```diff
@@ -1,199 +1,151 @@
-</KtPopover>
-<KtPopover size="xl" class="mt-4 ml-4">
-	<KtButton label="Extra Large Popover" />
-	<div slot="content">Message</div>
-</KtPopover>
-</div>
-
-## Scoped Slot
-
-Scoped slot allows your to get props or function provided by slot themselve.
-In KtPopover, `close` function is provided to allow user click a button from slot content and close the popover.
-
-<div class="element-example">
-	<KtPopover class="mt-4 ml-4">
-		<KtButton label="Close with Cancel Button" />
-		<div slot="content" slot-scope="slotProps">
-			<p>Save your message</p>
-			<KtButton type="text" @click="slotProps.close">Cancel</KtButton>
-			<KtButton type="primary">Save</KtButton>
-		</div>
-	</KtPopover>
-</div>
-
-```html
-<KtPopover>
-	<KtButton label="Close with Cancel Button" />
-	<div slot="content" slot-scope="slotProps">
-		<p>Save your message</p>
-		<KtButton type="text" @click="slotProps.close" label="Cancel" />
-		<KtButton type="primary" label="Save" />
-	</div>
-</KtPopover>
-```
-
-## Nested Select
-
-<div class="element-example">
-	<KtPopover placement="top" class="mt-4 ml-4">
-		<KtButton label="Test Popover with Dropdown"/>
-		<div slot="content">
+<template>
+	<div>
+		<ComponentInfo v-bind="{ component }" />
+
+		<p>
+			Use popovers to provide extra information or actions. Compared to
+			<code>tooltip</code>, <code>popovers</code> can carry more information.
+			<code>KtPopover</code> <strong>supports</strong> escaping the
+			<a
+				href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context"
+			>
+				CSS Stacking Context
+			</a>
+		</p>
+
+		<h2>Interactive Example</h2>
+
+		<KtForm v-model="values">
+			<KtFieldToggle
+				formKey="useOptions"
+				isOptional
+				label="use options"
+				type="switch"
+			>
+				<template #helpText>
+					Passing <code>options</code> turns <code>KtPopover</code> into a
+					pre-made dropdown button menu.
+					<dl>
+						<dt>TypeScript</dt>
+						<dd><code>Kotti.Popover.Props['options']</code></dd>
+					</dl>
+					<pre
+						v-text="JSON.stringify(exampleOptions, replacer, ' '.repeat(3))"
+					/>
+				</template>
+			</KtFieldToggle>
 			<KtFieldSingleSelect
-				label="Test with dropdown"
-				:options="[{ label: 'Click me', value: 'test_click' }]"
-				:value="null"
+				formKey="size"
+				hideClear
+				isOptional
+				label="size"
+				:options="sizeOptions"
 			/>
+			<KtFieldSingleSelect
+				formKey="placement"
+				hideClear
+				isOptional
+				label="placement"
+				:options="placementOptions"
+			>
+				<template #helpText>
+					There are 4 different positions for popovers:
+					<code>right</code>, <code>left</code>, <code>top</code>, and
+					<code>bottom</code>, or <code>auto</code>. All of which can be
+					appended with <code>-start</code> and <code>-end</code> to define
+					placement.
+					<br />
+					<a href="https://atomiks.github.io/tippyjs/v6/all-props/#placement"
+						>See Placement Options here</a
```

Comment:
Add a space before/after the words here to fix the autoformatting being overly restrictive